azure-pipelines.yml

```diff
@@ -3,17 +3,7 @@
 # Add steps that analyze code, save build artifacts, deploy, and more:
 # https://docs.microsoft.com/azure/devops/pipelines/languages/java
 
-strategy:
-  matrix:
-    linux:
-      imageName: 'ubuntu-16.04'
-    mac:
-      imageName: 'macOS-10.15'
-    windows:
-      imageName: 'vs2017-win2016'
-
-pool:
-  vmImage: $(imageName)
+pool: Default
```

Comment:
Think about matrix build using self-hosted agents. Also, I guess the name of the pool is 'Default' and a comment about what OS agent is used would be more informative

Reply:
A Windows Self hosted machine is used for the build. 
I will add the comments in the file.

Reply:
I think we need to discuss on whether we could use these self-hosted agents and if we don then definitely we need to run it on several OS as we do currently. changing this and merging it to master will affect all our current dev process and checks that it does on different os.  I suggest reserving this change to be merged in the master branch